test/unit/request-auth.test.js

```diff
@@ -106,6 +106,28 @@ describe('RequestAuth', function () {
         });
     });
 
+    describe('ntlm', function () {
+        it('should be able to load all parameters from a request', function () {
+            var data = {
+                    auth: {
+                        type: 'ntlm',
+                        ntlm: {
+                            username: 'testuser',
+                            password: 'testpass',
+                            domain: 'testdomain',
+                            workstation: 'sample.work'
+                        }
+                    },
+                    url: 'httpbin.org/get'
```

Comment:
Although this doesn't actually result in a request, it should point to `echo/get`